app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCPaneModel.scala

```diff
@@ -339,6 +340,41 @@ class DLCPaneModel(resultArea: TextArea, oracleInfoArea: TextArea)
                               this)
   }
 
+  def cancelDLC(status: DLCStatus): Unit = {
+    if (status.state == DLCState.Offered || status.state == DLCState.Accepted) {
```

Comment:
Please pattern match on the states so we have exhaustiveness warnings if we add a new `DLCState` in the future.